### DISCUSSION TITLE
Submission for your Laptop List: Lenovo ThinkPad T490

### DISCUSSION THREAD

**Discussion by 5T33Z0** [daliansky/Hackintosh#411] [General]
Hi.

I recently acquired a Lenovo T490. Here's the link to the repo, if you want to add it to the list: 

https://github.com/5T33Z0/Thinkpad-T490-Hackintosh-OpenCore

**Comment by aleelmaitro** [comment]
Excuse the question...
What can be found here...
I'm trying to make a simple query, and all the forums reject it...
I don't want support... I have an error in an app for Hackintosh and I just want to know how to solve it...
I hope my query is not a bother...
